project-minimization-plugin/src/test/kotlin/FileLevelStageTest.kt

```diff
@@ -21,8 +21,10 @@ class FileLevelStageTest : JavaCodeInsightFixtureTestCase() {
 
     override fun setUp() {
         super.setUp()
-        project.service<MinimizationPluginSettings>().state.currentCompilationStrategy = CompilationStrategy.DUMB
-        project.service<MinimizationPluginSettings>().state.minimizationTransformations.clear()
+        var compilationStrategy by project.service<MinimizationPluginState>().stateObservable.compilationStrategy.mutable()
+        compilationStrategy = CompilationStrategy.DUMB
+        val transformations by project.service<MinimizationPluginState>().stateObservable.minimizationTransformations.mutable()
+        transformations.clear()
```

Comment:
As I said previously, it's not working for now. You should use something like: `transformations = emptyList()`